apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -351,3 +377,8 @@ watch([sortBy, choiceFilter], () => {
     />
   </teleport>
 </template>
+<style scoped>
```

Comment:
We can avoid this style by using the class `h-[calc(100vh-113px)]`

Reply:
tq tq